iOS/Flows/Conversation/Cells/EmotionCircleCollectionViewLayout.swift

```diff
@@ -171,6 +171,26 @@ class EmotionCircleCollectionViewLayout: UICollectionViewLayout {
     }
 }
 
+extension EmotionCircleCollectionViewLayout: UICollisionBehaviorDelegate {
+
+    func collisionBehavior(_ behavior: UICollisionBehavior,
+                           beganContactFor item: UIDynamicItem,
+                           withBoundaryIdentifier identifier: NSCopying?,
+                           at p: CGPoint) {
+
+        let vector = CGVector(dx: item.center.x - p.x, dy: item.center.y - p.y)
+        let pushBehavior = UIPushBehavior(items: [item], mode: .instantaneous)
+        pushBehavior.pushDirection = vector
+        pushBehavior.magnitude = 0.05
+        pushBehavior.action = { [unowned self, unowned pushBehavior] in
+            // Clean up the push after it's done
+            guard !pushBehavior.active else { return }
+            self.animator.removeBehavior(pushBehavior)
+        }
+        self.animator.addBehavior(pushBehavior)
+    }
+}
```

Comment:
To keep the circles moving, I give them a little extra push when they hit the sides.